src/pal/prebuilt/inc/metahost.h

```diff
@@ -59,13 +59,6 @@ typedef interface ICLRMetaHostPolicy ICLRMetaHostPolicy;
 #endif 	/* __ICLRMetaHostPolicy_FWD_DEFINED__ */
 
 
-#ifndef __ICLRProfiling_FWD_DEFINED__
```

Comment:
I didn't notice metahost idl changes that I would expect? Changes in this header would be erased the next time it is regenerated without corresponding idl changes.